src/pages/wishlist.html

```diff
@@ -0,0 +1,193 @@
+                </div>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <div class="content-wrapper">
+            <h3 class="text-center">My Wishlist <span class="light-txt">(3 items)</span></h3>
+          <section id="wishlist" class="wishlist-wrapper">
+            <div class="card card-vertical products-list-item">
+              <div class="card-media">
+                <img
+                  src="../images/clothes/dgClothes-4.jpg"
+                  alt=""
+                  class="responsive-img"
+                  loading="lazy"
+                />
+              </div>
+              <div class="card-description-wrapper">
+                <div class="card-primary">
+                  <p class="uppercase">Skull Dog Tank Top</p>
+                  <p class="card-primary-subtitle">Rs. 899/-</p>
+                </div>
+                <div class="card-actions">
+                  <button class="btn primary-btn uppercase">Add to cart</button>
+                </div>
+                <button class="borderless-btn ml-8 wishlist-btn"><i class="far fa-heart fa-lg"></i></button>
+              </div>
+            </div>
+            <div class="card card-vertical products-list-item">
+              <div class="card-media">
+                <img
+                  src="../images/food/dgFood-6.png"
+                  alt=""
+                  class="responsive-img"
+                  loading="lazy"
+                />
+              </div>
+              <div class="card-description-wrapper">
+                <div class="card-primary">
+                  <p class="uppercase">Instant Bone Broth Chicken</p>
+                  <p class="card-primary-subtitle">Rs. 499/-</p>
+                </div>
+                <div class="card-actions">
+                  <button class="btn primary-btn uppercase">Add to cart</button>
+                </div>
+                <button class="borderless-btn ml-8 wishlist-btn"><i class="far fa-heart fa-lg"></i></button>
+
+              </div>
+            </div>
+            <div class="card card-vertical products-list-item">
+                <div class="card-media">
+                  <img
+                    src="../images/food/dgFood-2.jpg"
+                    alt=""
+                    class="responsive-img"
+                    loading="lazy"
+                  />
+                </div>
+                <div class="card-description-wrapper">
+                  <div class="card-primary">
+                    <p class="uppercase">Grain Free Chicken & Apple</p>
+                    <p class="card-primary-subtitle">Rs. 299/-</p>
+                  </div>
+                  <div class="card-actions">
+                    <button class="btn primary-btn uppercase">Add to cart</button>
+                  </div>
+                  <button class="borderless-btn ml-8 wishlist-btn"><i class="far fa-heart fa-lg"></i></button>
```

Comment:
As this is a wishlist page there is no point in having an add to wishlist icon here because the items are already in the wishlist. Add a remove from wishlist icon in place of add to wishlist icon.